website/src/docs/uppy.md

```diff
@@ -69,6 +70,14 @@ const photoUploader = Uppy({ id: 'post' })
 
 By default Uppy will wait for an upload button to be pressed in the UI, or an `.upload()` method to be called, before starting an upload. Setting this to `autoProceed: true` will start uploading automatically after the first file is selected.
 
+### `allowMultipleUploads: true`
+
+Whether to allow multiple upload batches. This means multiple calls to `.upload()`, or a user pressing an upload button multiple times, perhaps after adding more files. An upload batch is made up of the files that were added since the previous `.upload()` call.
+
```

Comment:
All makes sense when I read twice, but might still be a liiittle confusing, if one doesn’t know about Uppy’s internals, the option could be called `allowAddingMoreFilesAfterUploadStarted`, right? Not suggesting the crazy name, but trying to think how to indicate that this is what it means right away.